gatsby-browser.js

```diff
@@ -138,7 +138,9 @@ export const onClientEntry = () => {
     updateURLAsPerUserLanguage()
 }
 
-export const onRouteUpdate = () => {
+export const onRouteUpdate = ({ location }) => {
+    Analytics.pageView(location.pathname, 'Deriv.com')
```

Comment:
Hi @amir-deriv ,
Just asking is this fine capital `D` in `Deriv.com`

Reply:
Yes @fasihali-deriv , this is the convention we are following, for `deriv-app` we are using `Deriv App`

Reply:
Alright. Thank you so much 🙏